lib/facilities/client.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+require 'geoservices'
+require 'openssl'
+
+# TODO: PV: Obviously this is not useful, but maps.va.gov has a self-signed cert
+# so try this for testing purposes.
+OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
```

Comment:
See https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/evss/base_service.rb#L74:L100 for what we did for EVSS's cert.